src/main/scala/scalismo/statisticalmodel/DiscreteGaussianProcess.scala

```diff
@@ -63,13 +63,13 @@ class DiscreteGaussianProcess[D <: Dim: NDSpace, Value] private[scalismo] (val m
    * The marginal distribution for the points specified by the given point ids.
    * Note that this is again a DiscreteGaussianProcess.
    */
-  def marginal(pointIds: Seq[PointId])(implicit domainCreator: UnstructuredPointsDomain.Create[D]): DiscreteGaussianProcess[D, Value] = {
+  def marginal(pointIds: Seq[PointId])(implicit domainCreator: UnstructuredPointsDomain.Create[D]): DiscreteGaussianProcess[D, UnstructuredPointsDomain[D], Value] = {
```

Comment:
@marcelluethi Does this mean that when marginalizing a DiscreteLowRankGaussianProcess[_3D, DiscreteImageDomain[_3D], Double] we would always end up with DiscreteLowRankGaussianProcess[_3D, UnustructuredPointsDomain[_3D], Double] ? An efficient interpolation of the marginal later on would not be possible.